### PR TITLE
Adding translation for zh-cn (Simplified Chinese)

### PR DESCRIPTION
**Pull request by tigersoldier**
Note that I changed the language names to lookup from {lang}-{region} instead of just {lang}, because Genshin Impact supports both zh-cn and zh-tw locales. zh-tw's tranlsation is different zh-cn.

### CODE REVIEW

**Comment by Mar0xy** [comment]
There are three missing lines now as there were new ones added while you were working on this

I have linked them here
https://github.com/an-anime-team/an-anime-game-launcher/commit/309da61f4ff422dc778cb693d3545638386660a4#diff-abd63e9554c94e744d1960e66f066af5eef32276a58b827f00fecdeb1b9e929a
https://github.com/an-anime-team/an-anime-game-launcher/commit/309da61f4ff422dc778cb693d3545638386660a4#diff-24ac8d8faf07ad0d9a9e1c9f8af202a0534db84d231d3aeae186456f9a5ed3ec

File: assets/locales/zh-cn/enhancements.ftl [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+fps-unlocker = 解除帧率限制
+
+enabled = 开启
+fps-unlocker-description = 修改游戏内存，解除渲染帧率限制。可能会触发反作弊检查
+
+power-saving = 省电模式
+power-saving-description = 当游戏失去焦点时 (例如切出游戏) 自动当帧率设为 10, 并降低进程优先级
+
+monitor = 显示器
+monitor-description = Number of monitor you want to run the game on
```
Comment (krypt0nn) [inline assets/locales/zh-cn/enhancements.ftl:44]: Guess you missed this one?

Comment (tigersoldier) [reply assets/locales/zh-cn/enhancements.ftl]: Updated. Was going to confirm if the "number" is the index or count. Verified that it should be the index.

**Comment by tigersoldier** [comment]
> There are three missing lines now as there were new ones added while you were working on this
> 
> I have linked them here [309da61#diff-abd63e9554c94e744d1960e66f066af5eef32276a58b827f00fecdeb1b9e929a](https://github.com/an-anime-team/an-anime-game-launcher/commit/309da61f4ff422dc778cb693d3545638386660a4#diff-abd63e9554c94e744d1960e66f066af5eef32276a58b827f00fecdeb1b9e929a) [309da61#diff-24ac8d8faf07ad0d9a9e1c9f8af202a0534db84d231d3aeae186456f9a5ed3ec](https://github.com/an-anime-team/an-anime-game-launcher/commit/309da61f4ff422dc778cb693d3545638386660a4#diff-24ac8d8faf07ad0d9a9e1c9f8af202a0534db84d231d3aeae186456f9a5ed3ec)

Added them as well. In general is there a translation management tool for synchronizing new items?

**Comment by krypt0nn** [comment]
No such tool. I generally send new lines needed to translate to special channel in your discord server

**Comment by Mar0xy** [comment]
> > There are three missing lines now as there were new ones added while you were working on this
> > I have linked them here [309da61#diff-abd63e9554c94e744d1960e66f066af5eef32276a58b827f00fecdeb1b9e929a](https://github.com/an-anime-team/an-anime-game-launcher/commit/309da61f4ff422dc778cb693d3545638386660a4#diff-abd63e9554c94e744d1960e66f066af5eef32276a58b827f00fecdeb1b9e929a) [309da61#diff-24ac8d8faf07ad0d9a9e1c9f8af202a0534db84d231d3aeae186456f9a5ed3ec](https://github.com/an-anime-team/an-anime-game-launcher/commit/309da61f4ff422dc778cb693d3545638386660a4#diff-24ac8d8faf07ad0d9a9e1c9f8af202a0534db84d231d3aeae186456f9a5ed3ec)
> 
> Added them as well. In general is there a translation management tool for synchronizing new items?

I thought about setting up an tool like weblate or crowdin but then realized that it doesn't really work well cause none of them support Fluent nor does it work well for adding new languages due to the manual editing required for i18n.rs and the common.ftl file

**Pull request merged**